### PR TITLE
Enable customer multi-currency for all merchants

### LINKED ISSUE #2482: Remove the customer multi-currency feature flag
When we have confirmed that all is in order and are ready to launch we should remove the feature flag to make the multi-currency available to all merchants.

### PR DESCRIPTION
**Pull request by luizreis**
Fixes #2482 

#### Changes proposed in this Pull Request

Enable customer multi-currency for all merchants

This commit modifies the default value for the multi-currency feature flag option to '1'. This results in the feature being enabled for anyone that did not set it to be disabled.

This approach enables customer multi-currency for all merchants and allows us to easily disable the entire extension if anything goes wrong or if requested by some merchant.

#### Testing instructions

1. Remove the `_wcpay_feature_customer_multi_currency` option from your database with `npm run wp -- option delete _wcpay_feature_customer_multi_currency`
2. Navigate to the WooCommerce Settings page and assert that the menu item for Multi-Currency is not there
3. Apply this PR and assert that the menu item shows up and Multi-Currency is enabled

-------------------

- [x] Added changelog entry (or does not apply)
- [x] Covered with tests (or have a good reason not to test in description ☝️)
- [x] Tested on mobile (or does not apply)

### CODE REVIEW

File: changelog.txt [hunk 1]
```diff
@@ -1,6 +1,7 @@
 *** WooCommerce Payments Changelog ***
 
 = 2.8.0 - 2021-xx-xx =
+* Add - Enable customer multi-currency.
```
Comment (luizreis) [inline changelog.txt:4]: What should we use as the changelog entry?

cc @LevinMedia, @dwainm

Comment (dwainm) [reply changelog.txt:4]: I've added the changelog entry as: 

```
Add - Allow merchants to add additional currencies to their store, allowing a store’s customers to shop and browse in the currency of their choice.
```

Comment (dwainm) [reply changelog.txt:4]: I submitted it to the release process, but still open to change.

Comment (luizreis) [reply changelog.txt]: Cool, thanks! I've updated it to match that one in https://github.com/Automattic/woocommerce-payments/pull/2552/commits/a7aefe4f3b3d328e6ce9a3754a328a940c4d8624.

**Review by jessepearson — APPROVED** [review]
LGTM! Tested and Multi-Currency works with the option deleted in the database.

**Pull request merged**